CHANGELOG.md

```diff
@@ -14,6 +14,12 @@
 
 🆕 New features:
 
+- Support custom attributes on summary list action links
+
+  The `attributes: {}` pattern can now be used to add `id` or `data-attr` attributes to summary list action links.
```

Comment:
Somewhat minor, but this is oddly specific in that it makes it sound a little bit like attributes can only be used for `id` / `data-*` attributes, when actually it can be used for any attributes.

How about something like:

```suggestion
  You can now use the `attributes` macro option to add additional HTML attributes to summary list action links.
```

?

Reply:
Sounds good to me, I've added it